testsuite/MDAnalysisTests/analysis/test_nuclinfo.py

```diff
@@ -40,3 +40,11 @@ def test_wc_pair_1(self):
     def test_wc_pair_2(self):
         val = nuclinfo.wc_pair(self.u, 22, 23, seg1='RNAA', seg2='RNAA')
         assert_almost_equal(val, 4.601, decimal=3)
+
+    def test_minor_pair_1(self):
+        val = nuclinfo.minor_pair(self.u, 3, 17, seg1='RNAA', seg2='RNAA')
+        assert_almost_equal(val, 16.649, decimal=3)
```

Comment:
If it's regression then probably just from running the tests in filling in the values that are needed to make the tests pass.